packages/react-native/Libraries/Modal/__tests__/__snapshots__/Modal-test.js.snap

```diff
@@ -13,7 +13,7 @@ exports[`<Modal /> should render as <RCTModalHostView> when not mocked 1`] = `
 <RCTModalHostView
   animationType="none"
   hardwareAccelerated={false}
-  identifier={4}
```

Comment:
What caused this to change?

Reply:
I happens due to `Modal` keeping [auto-incrementing identifier](https://github.com/facebook/react-native/blob/dab7738a8865b5eb65208adb6930fa8211b220bf/packages/react-native/Libraries/Modal/Modal.js#L192) for each composite `Modal` instance. Since tests are not isolated, the removal of rendering of `Modal` when `visible={false}` in previous test reduced the count by 1.